tests/test_autoscaling/test_autoscaling.py

```diff
@@ -8,7 +8,8 @@
 import boto.ec2.elb
 import sure  # noqa
 
-from moto import mock_autoscaling, mock_ec2_deprecated, mock_elb_deprecated, mock_autoscaling_deprecated, mock_ec2
+from moto import mock_autoscaling, mock_ec2_deprecated, mock_elb_deprecated, mock_autoscaling_deprecated, \
```

Comment:
If you're cleaning this line up there's probably a clearer way these imports could be expressed. IMO they should be all on their own lines for readability